planet/cli/subscriptions.py

```diff
@@ -192,11 +197,12 @@ def request(name, source, delivery, notifications, tools, pretty):
     echo_json(res, pretty)
 
 
-@subscriptions.command()
+@subscriptions.command(epilog=valid_item_string)
+@translate_exceptions
 @click.option('--item-types',
               required=True,
-              type=types.CommaSeparatedString(),
-              help='One or more comma-separated item types.')
+              help='Item type for requested item ids.',
+              type=types.CommaSeparatedString())
```

Comment:
validation of item types should be performed here, see https://github.com/planetlabs/planet-client-python/blob/a345e0229094d24885be79d33da496be1172bed9/planet/cli/data.py#L327 for an example.